packages/twenty-server/src/modules/charts/standard-objects/chart.workspace-entity.ts

```diff
@@ -0,0 +1,126 @@
+import { Relation } from 'typeorm';
+
+import { FieldMetadataType } from 'src/engine/metadata-modules/field-metadata/field-metadata.entity';
+import { CHART_STANDARD_FIELD_IDS } from 'src/engine/workspace-manager/workspace-sync-metadata/constants/standard-field-ids';
+import { STANDARD_OBJECT_IDS } from 'src/engine/workspace-manager/workspace-sync-metadata/constants/standard-object-ids';
+import { WorkspaceEntity } from 'src/engine/twenty-orm/decorators/workspace-entity.decorator';
+import { WorkspaceField } from 'src/engine/twenty-orm/decorators/workspace-field.decorator';
+import { WorkspaceIsNullable } from 'src/engine/twenty-orm/decorators/workspace-is-nullable.decorator';
+import { BaseWorkspaceEntity } from 'src/engine/twenty-orm/base.workspace-entity';
+import { WorkspaceIsSystem } from 'src/engine/twenty-orm/decorators/workspace-is-system.decorator';
+import {
+  RelationMetadataType,
+  RelationOnDeleteAction,
+} from 'src/engine/metadata-modules/relation-metadata/relation-metadata.entity';
+import { WorkspaceRelation } from 'src/engine/twenty-orm/decorators/workspace-relation.decorator';
+import { ChartFilterWorkspaceEntity } from 'src/modules/charts/standard-objects/chart-filter.workspace-entity';
+
+@WorkspaceEntity({
+  standardId: STANDARD_OBJECT_IDS.chart,
+  namePlural: 'charts',
+  labelSingular: 'Chart',
+  labelPlural: 'Charts',
+  description: 'A chart for data visualization',
+  icon: 'IconChartBar',
+})
+export class ChartWorkspaceEntity extends BaseWorkspaceEntity {
+  @WorkspaceField({
+    standardId: CHART_STANDARD_FIELD_IDS.name,
+    type: FieldMetadataType.TEXT,
+    label: 'Name',
+    description: 'Chart name',
+    icon: 'IconNotes',
+  })
+  name: string;
```

Comment:
FYI ongoing PR to set the label identifier via a decorator https://github.com/twentyhq/twenty/pull/6227 ; should be merged today probably